images/ova/flatcar.yaml

```diff
@@ -0,0 +1,26 @@
+---
+download_images: true
+build_name: "flatcar-3033.3.16"
+packer_builder_type: "vsphere"
+guestinfo_datasource_slug: "https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo"
+guestinfo_datasource_ref: "v1.4.0"
+guestinfo_datasource_script: "{{guestinfo_datasource_slug}}/{{guestinfo_datasource_ref}}/install.sh"
+packer:
+  cluster: ""
+  datacenter: ""
+  datastore: ""
+  folder: ""
+  insecure_connection: "false"
+  network: ""
+  resource_pool: ""
+  template: "flatcar_production_lts_release_ova"
```

Comment:
I guess lets wait for the "correct" template here?

Reply:
I dont have access to the `base-d2iq-templates` folder. looks like it will take until next week to get base template up there. We can leave this PR open until we get the base template up.

Reply:
Updated base templates from `base-d2iq-templates` https://github.com/mesosphere/konvoy-image-builder/pull/880/commits/9f6caea9a625ced742007a3463d961c5374006da